app/js/index.js

```diff
@@ -0,0 +1,307 @@
+/*jslint plusplus: true */
```

Comment:
Should be some kind of a module. Please read
https://toddmotto.com/mastering-the-module-pattern/